apis/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -1,4 +1,3 @@
-//go:build !ignore_autogenerated
```

Comment:
Why is this removed?

Reply:
It was generated automatically....

Reply:
I can remove it from the PR if it is necessary